ui/app/templates/vault/cluster/init.hbs

```diff
@@ -95,15 +95,14 @@
             </div>
           {{/if}}
           <DownloadButton
-            class="button is-ghost"
```

Comment:
<img width="452" alt="Screenshot 2023-09-28 at 4 54 39 PM" src="https://github.com/hashicorp/vault/assets/68122737/0df0abf4-22c0-4c61-835d-473ca128d504">

Reply:
Purposefully not matching css 1:1 with our old styling.

I'd prefer to make adjustments globally after we've removed the `.button` class, while consulting design, so we can align with other products and rely on HDS styling as much as possible